src/packet/tcp.rs.in

```diff
@@ -99,17 +120,60 @@ impl TcpOption {
     }
 
     /// Create Timestamp option
```

Comment:
It would be quite nice to get a little more description in the documentation comments - if I'm not familiar with TCP I should be able to figure out what the option does by looking at the doc comments.